### PR TITLE
first go

### PR DESCRIPTION
**Pull request by womullan**
Have a look and see if i caught the points ..

### CODE REVIEW

File: RTN-003.tex [hunk 1]
```diff
@@ -27,7 +27,7 @@
...
+This document provides guidelines for groups that are independent from the Rubin Project and Operations (i.e. US Data Facility) and would like to stand up an independent Data Access Center (IDAC; existing data centers that could serve Rubin data products are considered IDACs for purposes of this document). Some IDACs may want to serve only a subset of the  data products: this document proposes three portion sizes, from full releases to a "lite" catalog without posteriors. Guidelines and requirements for IDACs in terms of data storage, computational resources, dedicated personnel, and user authentication are described, as well as a preliminary assessment of the cost impacts. Some institutions, even those inside the US and Chile, may serve LSST data products locally to their research community. Requirements and responsibilities for such institutional bulk data transfers are also described here.
 }
```
Comment (leannep) [inline RTN-003.tex:31]: I am not a fan of using the 'lite' variant of the work light. Modern use typically refers to low-calorie foods or is used in branding  In traditional (middle english) use, it means "few" or "a little"

File: dataproducts.tex [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
...elease} catalogs, and not the images, is sufficient for the scientific needs of its community. This will probably require the specific \RO database server \citedsp{LDM-135} and specific machines, and the deployment of the database system and the associated subset of data access services (\gls{DAX}; e.g., web APIs, \gls{Qserv}, \citeds{LDM-152}). The full {\tt Object} catalog, which contains one row per object with a volume of $\approx 20$ kilobytes per row, is estimated to contain about $40 \times 10^9$ objects (even in the first full-sky data release). Adding to this the full {\tt Source} and {\tt Forced Source} catalogs, which contain one row per measurement in each of the $\sim80$ visit images obtained per year, brings the total storage volume required up into the petabytes range, and will require a serious commitment of resources at the proposed \gls{IDAC}. The evolution of catalog sizes over the 10-year \gls{LSST} is depicted in \figref{fig:catvol}, from which it is evident that the catalog size for the final release is order $15$ petabytes. For more details on the row counts see the Key Numbers Page\footnote{\url{https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers}}.
```
Comment (leannep) [inline dataproducts.tex:18]: I have submitted a ticket to remove that confluence page from public view. The subsystem scientists plan to do some more work on it. The page that should be referenced is the public facing one: https://www.lsst.org/scientists/keynumbers

**Review by leannep — APPROVED** [review]

File: dataproducts.tex (hunk 2 above)
Comment (womullan) [reply dataproducts.tex:18]: ok thats easy

File: RTN-003.tex (hunk 1 above)
Comment (womullan) [reply RTN-003.tex:31]: as discussed on slack this stays for now

**Pull request merged**